app/views/shell/_top.html.erb

```diff
@@ -63,6 +56,7 @@
         data-deployed-at="<%= ForemInstance.deployed_at %>"
         data-latest-commit-id="<%= ForemInstance.latest_commit_id %>"
         data-ga-tracking="<%= SiteConfig.ga_tracking_id %>">
+        <%= render "layouts/styles" %>
```

Comment:
This is the main functionality of this patch. This partial is now in the `head` _and_ the top of `body`.